src/simplewallet/simplewallet.cpp

```diff
@@ -3648,6 +3648,21 @@ static std::string get_human_readable_timestamp(uint64_t ts)
   return std::string(buffer);
 }
 //----------------------------------------------------------------------------------------------------
+static std::string get_human_readable_timespan(uint64_t ts)
+{
+  if (ts < 60)
+    return std::to_string(ts) + tr(" seconds");
+  if (ts < 3600)
+    return std::to_string((int)(ts / 60)) + tr(" minutes");
+  if (ts < 3600 * 24)
+    return std::to_string((int)(ts / 3600)) + tr(" hours");
+  if (ts < 3600 * 24 * 30.5)
+    return std::to_string((int)(ts / (3600 * 24))) + tr(" days");
+  if (ts < 3600 * 24 * 365.25)
+    return std::to_string((int)(ts / (3600 * 24 * 365.25))) + tr(" months");
```

Comment:
I meant that OSes just use days as the largest time interval since months and years are funky.